packages/react-components/theme-designer/src/components/Export/ExportPanel.tsx

```diff
@@ -46,26 +46,85 @@ const useStyles = makeStyles({
     height: '100%',
     boxSizing: 'border-box',
   },
+  panelContainer: {
+    zIndex: 100,
+    position: 'absolute',
+    top: '0px',
+    right: '0px',
+    width: '400px',
```

Comment:
I know this is not your code, but in the interest of wax on, wax off....

I like to define file level semantic constants anytime a value gets to be significantly larger than the general tokens. And I often define them for things smaller than that as well.

This helps future readers understand where this "Big random number" came from, but more importantly that the decision was carefully considered and coordinated. The name leaves a breadcrumb trail of how to get there.

Reply:
again, NYC, but same idea with z-indexes.